Fabric.Authorization.AccessControl/src/app/services/global/client-access-control-config.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { AuthService } from './auth.service';
+
+@Injectable()
+export class ClientAccessControlConfigService {
+
+  constructor(private authService: AuthService) { }
+
+  clientId = 'fabric-angularsample';
+
+  getAccessToken(){
+     return this.authService.getUser()
+    .then(function(user){           
+    if(user){
+         return Promise.resolve(user.access_token);
```

Comment:
you dont have to create another promise here, it will work just fine if you `return user.access_token;`

Reply:
ahh yeah you are correct. i will remove that extra promise